english-spanish/Part 2/english-spanish-Part 2-Family 2.csv

```diff
@@ -1,11 +1,18 @@
-italiano,Italian
-su,his
+italiano,Italian (m)
+italiana, Italian (f)
+su,his/her
+hija,daughter
```

Comment:
this one is in Part1 family and I wanted to avoid duplicates throughout the whole program